ports/geckolib/gecko_bindings/tools/regen.py

```diff
@@ -275,7 +280,16 @@ def build(objdir, target_name, kind_name=None,
         for ty in current_target["void_types"]:
             flags.append("-raw-line")
             flags.append("pub enum {} {{}}".format(ty))
-
+    if "refptr_types" in current_target:
```

Comment:
refptr_types is the wrong name IMO, This lists the types which are actually Arc<T> on the servo side. Perhaps servo_arc_types would be better?